awx/lib/awx_display_callback/module.py

```diff
@@ -308,8 +308,10 @@ def v2_playbook_on_stats(self, stats):
             changed=stats.changed,
             dark=stats.dark,
             failures=stats.failures,
+            ignored=stats.ignored,
```

Comment:
Wherever we land on this, we *also* need to get this into `ansible-runner` here:

https://github.com/ansible/ansible-runner/blob/master/ansible_runner/display_callback/module.py#L300

...so we don't regress down the road.

Reply:
good point, this issues applies to runner just as it does to AWX. Hopefully that's the extent of the changes needed?

Reply:
I expect that it should be, as this is the only change to `awx/lib/awx_display_callback` in runner (and the version in runner hasn't changed any that I can tell).

Reply:
@ryanpetrello - Updated runner here:
https://github.com/ansible/ansible-runner/pull/215